dialogue/TownNPCs/Mocha/InitialInteractions/Azura/Azura_Mocha_First2.json

```diff
@@ -20,12 +20,19 @@
       "Bueno, ¡yo qué sé!'.\n¡Estoy muy ocupada!",
       "Me aseguraré de que nadie entre en tu habitación mientras tanto.."
     ],
+    "pl": [
+      "Kurcze, więc udało mu się uciec, ha... Miasteczko w nieładzie oznacza złe wieści dla moich rezerwacjach...",
```

Comment:
I think that "co" will be better than "ha" in this case. Also you should use "rezerwacji" instead of "rezerwacjach"